publish/samplesite/instrumentation-tests/__instrumentation-tests__/customReplacePlugin.test.ts

```diff
@@ -20,100 +20,133 @@ describe('Editor', () => {
+                    replacementHTML: '<_<',
+                    matchSourceCaseSensitive: false,
+                },
+            ])
+        );
         await focusEditor(page);
         await page.keyboard.type('this is a source string with spaces!!');
 
         // Act
-        const resultContent: string = await page.evaluate(() => (window as any).globalRoosterEditor.getTextContent());
-        expect(resultContent.trim()).toEqual("<_<!!");
+        const resultContent: string = await page.evaluate(() =>
+            (window as any).globalRoosterEditor.getTextContent()
+        );
+        expect(resultContent.trim()).toEqual('<_<!!');
     });
 
-
     it('Should not match case other replacements on case sensitive matches', async () => {
         // Arrange
-        await page.evaluate(() => (window as any).editorPlugins.customReplace.updateReplacements([{
-            sourceString: 'Hello',
-            replacementHTML: '👋',
-            matchSourceCaseSensitive: true,
-        }]));
+        await page.evaluate(() =>
+            (window as any).editorPlugins.customReplace.updateReplacements([
+                {
+                    sourceString: 'Hello',
+                    replacementHTML: '👋',
+                    matchSourceCaseSensitive: true,
+                },
+            ])
+        );
         await focusEditor(page);
         await page.keyboard.type('hello');
 
         // Act
-        const resultContent: string = await page.evaluate(() => (window as any).globalRoosterEditor.getTextContent());
-        expect(resultContent.trim()).toEqual("hello");
+        const resultContent: string = await page.evaluate(() =>
+            (window as any).globalRoosterEditor.getTextContent()
+        );
+        expect(resultContent.trim()).toEqual('hello');
     });
 
     it('Should match case insensitive source strings on case insensitive matches', async () => {
         // Arrange
-        await page.evaluate(() => (window as any).editorPlugins.customReplace.updateReplacements([{
-            sourceString: 'Hello',
-            replacementHTML: '👋',
-            matchSourceCaseSensitive: false,
-        }]));
+        await page.evaluate(() =>
+            (window as any).editorPlugins.customReplace.updateReplacements([
+                {
+                    sourceString: 'Hello',
+                    replacementHTML: '👋',
+                    matchSourceCaseSensitive: false,
+                },
+            ])
+        );
         await focusEditor(page);
         await page.keyboard.type('hello');
 
         // Act
-        const resultContent: string = await page.evaluate(() => (window as any).globalRoosterEditor.getTextContent());
-        expect(resultContent.trim()).toEqual("👋");
+        const resultContent: string = await page.evaluate(() =>
+            (window as any).globalRoosterEditor.getTextContent()
+        );
+        expect(resultContent.trim()).toEqual('👋');
     });
 });
```

Comment:
Not sure how this wasn't autoformatted before. The only actual change is renaming the `describe` block